pkg/plugin/plugin.go

```diff
@@ -143,9 +145,67 @@ func (p *Plugin) CmdAdd(args *skel.CmdArgs) error {
 		return fmt.Errorf("failed to save PluginConf %q", err)
 	}
 
+	if err = p.updateDeviceInfo(cmdCtx); err != nil {
+		log.Error().Msgf("failed to update DeviceInfo %v.", err)
+		// this step is not critical for CNI operation, log error and continue
+	}
 	return types.PrintResult(cmdCtx.result, current.ImplementedSpecVersion)
 }
 
+// updateDeviceInfo updates CNIDeviceInfoFile file with information
+// about VF representor
+func (p *Plugin) updateDeviceInfo(cmdCtx *cmdContext) error {
+	if cmdCtx.pluginConf.RuntimeConfig.CNIDeviceInfoFile == "" {
+		return nil
+	}
+	pciDevInfoKey := "pci"
+
+	devInfo := map[string]interface{}{}
+
+	bytes, err := os.ReadFile(cmdCtx.pluginConf.RuntimeConfig.CNIDeviceInfoFile)
+	if err != nil {
+		return fmt.Errorf("failed to read CNIDeviceInfoFile %q", err)
+	}
+	err = json.Unmarshal(bytes, &devInfo)
+	if err != nil {
+		return fmt.Errorf("failed to unmarshal CNIDeviceInfoFile %q", err)
+	}
+	pciInfo, exist := devInfo[pciDevInfoKey]
+	if !exist {
+		pciInfo = map[string]interface{}{}
+	}
+	pciInfoMap, ok := pciInfo.(map[string]interface{})
+	if !ok {
+		return fmt.Errorf("unexpected pci info format in CNIDeviceInfoFile")
+	}
+
+	setDevInfoPCIField(pciInfoMap, "pci-address", cmdCtx.pluginConf.DeviceID)
```

Comment:
Thats already provided by device plugin / multus no ?

Reply:
Yes, it is. As agreed above we will add only "representor-name" field in the bridge plugin